flang/test/Lower/namelist.f90

```diff
@@ -46,6 +46,28 @@ program p
   ! CHECK: fir.call @_FortranAioEndIoStatement([[cookie]]
   write(*, nnn)
 end
+
+! CHECK-LABEL: sss
+subroutine sss
+  integer xxx(11:13)
+  namelist /rrr/ xxx
+  ! CHECK: [[cookie:%[0-9]+]] = fir.call @_FortranAioBeginExternalListInput
+  ! CHECK: alloca
+  ! CHECK: undefined
+  ! CHECK: fir.address_of
+  ! CHECK: fir.insert_value
+  ! CHECK: fir.shape_shift
+  ! CHECK: fir.embox
+  ! CHECK: fir.insert_value
+  ! CHECK: fir.alloca
+  ! CHECK: fir.undefined
+  ! CHECK: fir.address_of
+  ! CHECK-COUNT-3: fir.insert_value
```

Comment:
These check lines are pretty lean.

Reply:
Yes; I think the primary ops in the sequence are the main thing to check, absent end-to-end execution results.  I've added a little more detail though.